code/modules/mob/living/simple_animal/hostile/commanded/guard_dog.dm

```diff
@@ -128,4 +128,44 @@
 
 	harm_intent_damage = 5
 	melee_damage_lower = 5
-	melee_damage_upper = 5
\ No newline at end of file
+	melee_damage_upper = 5
+
+
+
+/mob/living/simple_animal/hostile/commanded/dog/iangang
+	name = "Hood Rat Ian"
+	short_name = "hoodian"
+	desc = "Ian's Distant Cousin, however unlike Ian he only likes white girls and guns and money."
+	icon_state = "iangang"
+	icon_living = "iangang"
+	icon_dead = "iangang_dead"
+	health = 95
+	maxHealth = 95
+	melee_damage_lower=6
+	melee_damage_upper=11
+	name_changed = 1
+	known_commands = list("stay", "stop", "attack", "follow")
+	attacktext = "licked"
+	attack_sound = 'sound/misc/dog_bark.ogg'
+	faction = "syndicate"
+
+/mob/living/simple_animal/hostile/commanded/dog/iangang/listen(var/mob/speaker, var/text)
```

Comment:
That entire proc can be deleted, since we dont want to modify the commands from the parent.